src/mono/mono/mini/interp/interp.c

```diff
@@ -3963,6 +3963,39 @@ mono_interp_exec_method (InterpFrame *frame, ThreadContext *context, FrameClause
 				memmove (frame->stack, (guchar*)frame->stack + params_offset, params_size);
 				new_method = (InterpMethod*)frame->imethod->data_items [ip [2]];
 
+				{
```

Comment:
This should be done in transform.c i think, the only time this needs to happen in interp.c is when doing indirect calls.

Reply:
Fixed.